kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/WebsiteService.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.good.patterns.challenges;
+
+public class WebsiteService implements InformationService
```

Comment:
jeśli ta klasa ma być immutable to sama w sobie też powinna być opatrzona słowem kluczowym final

Reply:
Poprawione